apps/delivery-options/private/skipCssPlugin.ts

```diff
@@ -0,0 +1,12 @@
+import {type Plugin} from 'vite';
+
+export const skipCssPlugin = (): Plugin => ({
```

Comment:
deze is verplaatst en hernoemd, enige echte wijzigingen zijn dat ik `scss` heb vervangen door `css` in de naam, en dat hij hieronder ook `id.endsWith('.css')` heb toegevoegd